packages/paste-core/components/combobox/__tests__/index.spec.tsx

```diff
@@ -277,6 +317,22 @@ describe('Combobox', () => {
       // @ts-ignore Property 'value' does not exist on type 'HTMLElement' (I get it, but this is right)
       expect(screen.getByRole('textbox').value).toEqual('Alert');
     });
+
+    it('should not run react virtual\'s "scroll to" function when "groupItemsBy" is defined as a string', () => {
+      // bc grouped comboboxes are not yet virtualized
+      render(<GroupedMockCombobox />, {wrapper: ThemeWrapper});
+
+      const targetIndex = 1;
+      const target = groupedItems[targetIndex];
+
+      userEvent.hover(screen.getByText(target.label));
+
+      expect(screen.getByRole('textbox').getAttribute('aria-activedescendant')).toMatch(
+        /downshift-([1-9]\d\d|[1-9]\d|\d)-item-1/g
+      );
+
+      expect(mockScrollToIndex).not.toHaveBeenCalled();
```

Comment:
✅ I verified that this expect statement fails when that guard is removed (meaning the scroll function is in fact called).